spark/v3.5/spark-extensions/src/main/scala/org/apache/spark/sql/execution/datasources/v2/ExtendedDataSourceV2Strategy.scala

```diff
@@ -123,6 +128,18 @@ case class ExtendedDataSourceV2Strategy(spark: SparkSession) extends Strategy wi
         allowExisting = allowExisting,
         replace = replace) :: Nil
 
+    case d@DescribeRelation(ResolvedV2View(catalog, ident), _, isExtended, _) =>
```

Comment:
I understand not wanting to bother with `output`, but I think the best practice is to preserve `output` from the logical plan to the physical plan. That's because resolution has occurred with the IDs from the logical plan and we don't want to reassign internal IDs in the physical plan. They should always match. That's why most Spark plans (like `DescribeTableExec`) copy and reuse the output attrs.